starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +110,20 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(words, searchWord){
+  for (var i = 0; i< words.length; i++){
+    if (searchWord === words[i]){
+    return true;
+    } else {
+    return false;
+    }
+    break;
```

Comment:
This code only will work properly if the word is in first place. Either way, will return false before iterating all over the array (because line 118). Furthermore, the break does nothing. This is a possible solution:
```
function doesWordExist(words, searchWord) {
  for (var i = 0; i < words.length; i++) {
    if (words[i] === searchWord) {
      return true;
    }
  }
  return false;
}
```